test/unit/auth/auth-api-request.spec.ts

```diff
@@ -1761,12 +1764,23 @@ AUTH_REQUEST_HANDLER_TESTS.forEach((handler) => {
           .then(() => {
             throw new Error('Unexpected success');
           }, (error) => {
-            expect(error).to.deep.equal(expectedError);
+            expect(error).to.deep.include(expectedError);
             expect(stub).to.have.been.calledOnce.and.calledWith(
               callParams(path, method, userImportBuilder.buildRequest()));
           });
       });
 
+      function expectUserImportResult(result: UserImportResult, expected: UserImportResult): void {
```

Comment:
Should this be the function exported from `user-import-builder.spec.ts`?

Reply:
I think I forgot to push a local change. This should be gone now.